### DISCUSSION TITLE
essay/notes/2023-08-12

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#144] [Announcements]
# essay/notes/2023-08-12

偶然发现域名快到期了，花费261元直接再续9年域名（拉满）

https://sugarat.top/essay/notes/2023-08-12.html

**Comment by i-mufeng** [comment]
可以用 `acme.sh` 搞个[泛域名证书部署](https://www.imufeng.cn/technology/web/acme-sh.html)，虽然时间短，但是它可以自动更新。

**Reply by ATQQ** [reply]
Let's Encrypt 我有在用，只用在普通web上没问题，但小程序场景不行，不信任这个机构 用不了

**Reply by i-mufeng** [reply]
学到了学到了。

**Comment by cvpv** [comment]
服务器我买了阿里云99一年可以每年续费到2026年，不错，勉强够用，就是3mpbs带宽有点小。
博主你多少带宽？


昨天我也在琢磨换域名，
想申请个尽量短点的com域名
发现四位纯字母已经没了
只能申请四位字母加数字

我搞了个zc3q.com
哈哈，因为我名字叫zc，然后3q=栓Q。对于其他人来说估计也是乱码

**Reply by ATQQ** [reply]
我现在这个贼便宜300多 21年双十一买的腾讯云服务器 4核4G8M带宽
